src/fwupdmgr.md

```diff
@@ -0,0 +1,44 @@
+% fwupdmgr(1) @PACKAGE_VERSION@ | fwupdmgr man page
+
+NAME
+====
+
+**fwupdmgr** — firmware update manager client utility
+
+SYNOPSIS
+========
+
+| **fwupdmgr** [CMD]
+
+DESCRIPTION
+===========
+
+fwupdmgr is a command line fwupd client intended to be used interactively.
+The output between versions of fwupd is not guaranteed to be stable.
+
+OPTIONS
+=======
+
+The fwupdmgr command takes various options depending on the action.
+Run **fwupdmgr \-\-help** for the full list.
+
+EXIT STATUS
+===========
+
+Commands that successfully execute will return "0", but commands that have no
+actions but successfully execute will return "2".
+
+BUGS
+====
+
+See GitHub Issues: <https://github.com/fwupd/fwupd/issues>
+
+AUTHOR
+======
+
+Richard Hughes <richard@hughsie.com>
+
+SEE ALSO
+========
+
+fwupdagent(1), fwupdtool(1)
```

Comment:
This should be dynamic based whether `compat_cli` is set or not

Reply:
i think if the long term plan is to remove them then we shouldn't reference them at all.